e2e-tests/playwright/support/pages/CatalogImport.ts

```diff
@@ -55,20 +55,26 @@ export class BackstageShowcase {
     return response.json();
   }
 
+  async checkEnabledAndClick(locator: string) {
+    if (await this.page.locator(locator).isEnabled()) {
```

Comment:
In case of button not enabled due to loading or some reason, it will skip clicking on the button, do you wanted to wait until Enabled here?

Reply:
I want to click on the button only if it is enabled. After disabling the `Click on the arrows...` test, the following one is attempting `clickFirstPage` which fails because the button is disabled. This check that I'm adding is making sure the button is enabled = clickable.